src/rv32_ucode/dpath.scala

```diff
@@ -142,7 +142,6 @@ class DatPath(implicit val conf: SodorConfiguration) extends Module
    csr.io.rw.cmd   := io.ctl.csr_cmd
    csr_rdata       := csr.io.rw.rdata
    csr.io.retire    := io.ctl.upc_is_fetch
-   // for now, the ucode does NOT support exceptions
    csr.io.exception := io.ctl.exception //false.B
```

Comment:
Do you still need the commented out `false.B`?

Reply:
Fixed.